charts/datree-admission-webhook/templates/namespace-post-delete.yaml

```diff
@@ -4,7 +4,8 @@ metadata:
   name: datree-cleanup-namespaces-hook-pre-delete
   labels: {{include "datree.labels" . | nindent 4}}
   namespace: {{template "datree.namespace" .}}
-  annotations: 
+  annotations:
+    argocd.argoproj.io/hook: Skip
```

Comment:
because argocd doesn't have uninstall hook these resources should be skipped - ok by product